metadata.json

```diff
@@ -23,6 +23,10 @@
     {
       "name": "puppet/archive",
       "version_requirement": ">= 1.0.0 < 9.0.0"
+    },
+    {
```

Comment:
Please don't add the core modules here. They are always part of the AIO packages, so they are always present.

Reply:
I removed it and now the tests are failing with `Unknown resource type: 'augeas'`

Reply:
or do I need to leave in .fixtures?

Reply:
Maybe it was this: https://github.com/OpenVoxProject/puppet-runtime/pull/85

Reply:
Yep, it should be in the fixtures file, which is used for unit tests.